### DISCUSSION TITLE
How to scroll down with CTRL-E like in vim ?

### DISCUSSION THREAD

**Discussion by toyamarinyon** [zed-industries/zed#6500] [Feature Requests]
Hi,

I want to scroll down with CTRL-E like in vim.
https://vimdoc.sourceforge.net/htmldoc/scroll.html#CTRL-E

Any suggestions?

**Comment by notpeter** [comment]
Our vim mode supports using `ctrl-e` / `ctrl-y` to scroll up and down is this not working for you?

https://github.com/zed-industries/zed/blob/dd9b2e2cdeb0d09c2f0358d0a4b899098a6d901a/assets/keymaps/vim.json#L86-L87

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
